cmd/agent/main.go

```diff
@@ -42,33 +43,72 @@ const (
 )
 
 func main() {
-	if _, err := reconcileKubelet(context.Background()); err != nil {
-		logrus.Warnf("failed to reconcile kubelet, error: %v", err)
-	}
+	var err error
 
-	logrus.SetOutput(colorable.NewColorableStdout())
-	logserver.StartServerWithDefaults()
-	if os.Getenv("CATTLE_DEBUG") == "true" || os.Getenv("RANCHER_DEBUG") == "true" {
-		logrus.SetLevel(logrus.DebugLevel)
-	}
+	if len(os.Args) > 1 {
+		err = runArgs()
+	} else {
+		if _, err := reconcileKubelet(context.Background()); err != nil {
```

Comment:
Are we shadowing the error here?
```suggestion
		if _, err = reconcileKubelet(context.Background()); err != nil {
```

Reply:
ya good call this should have no colon